README.md

```diff
@@ -3,18 +3,47 @@ A list of resources for data science work on climate change
 
 ## Data Sets
 
+### Global
+* [ICE Embodied Carbon Database](https://circularecology.com/embodied-carbon-footprint-database.html)
+* [Carbon in food downloadable csv](https://ourworldindata.org/food-choice-vs-eating-local)
+* [EPW Weather files (for Ladybug on Grasshopper, but adaptable)](https://www.ladybug.tools/epwmap/)
+* [Renewable Ninja - front end for NASA dataset](https://www.renewables.ninja/)
+* [One of many NASA datasets. This one covers precipitation](https://disc.gsfc.nasa.gov/datasets/GPM_3IMERGHH_06/summary?keywords=IMERG)
+
+* [ Keen cyclist route heatmap](https://www.strava.com/heatmap)
+* [Havard atlas of economic complexity downloads](https://atlas.cid.harvard.edu/data-downloads)
+* [Open Street map API](https://operations.osmfoundation.org/policies/api/)
+
+
+### National
+
 * [UK Environmental Performance Certificates API](https://epc.opendatacommunities.org/docs/api)
+* [UK Live csv tables on EPCs that exist](https://www.gov.uk/government/statistical-data-sets/live-tables-on-energy-performance-of-buildings-certificates)
+* [UK Gridwatch datasets](https://gridwatch.co.uk/downloads)
+* [UK taded carbon values](https://www.gov.uk/government/collections/carbon-valuation--2)
+* [List of National level 2050 enrgy opathway calculators (scroll down a bit)](https://www.gov.uk/guidance/international-outreach-work-of-the-2050-calculator)
+
+### City
+
+* [Bogota Air Quality city data](http://influxdb.canair.io:8000/d/xRQpZACWk/static-stations?orgId=1&refresh=10s)
+
+
 
 ## Companies
 
 * [appsilon AI for Good](https://appsilon.com/ai-for-good/)
 * [Open Climate Fix](https://openclimatefix.org/)
+* [ICAX - good for seasonal heat trasnfer info](https://www.icax.co.uk/Getting_Serious_about_Net_Zero.html)
+
 
 ## Packages
+* [ Ladybug tools for Python (and grasshopper/dynamo](https://github.com/ladybug-tools)
```

Comment:
Great, let's keep this link and remove the one at the top of the doc.

Reply:
different things